apps/management/package.json

```diff
@@ -13,15 +13,15 @@
     "build:clay": "NEXT_PUBLIC_CERAMIC_NETWORK=testnet-clay npm run build && rm -rf ../../public-clay/management && mkdir -p ../../public-clay && mv ./out ../../public-clay/management"
   },
   "dependencies": {
-    "3id-did-provider": "^1.0.2",
+    "3id-did-provider": "^1.1.0",
     "@3id/manager": "^0.2.0",
     "@3id/window-auth-provider": "^0.2.0",
-    "@ceramicnetwork/blockchain-utils-linking": "^1.0.0",
-    "@ceramicnetwork/http-client": "^1.0.0",
+    "@ceramicnetwork/blockchain-utils-linking": "^1.1.2",
+    "@ceramicnetwork/http-client": "^1.2.0",
     "@ceramicnetwork/transport-postmessage": "^0.3.0",
-    "@ceramicstudio/idx": "^0.12.0",
-    "@ceramicstudio/multiauth": "^0.2.2",
-    "caip": "^0.9.2",
+    "@ceramicstudio/idx": "^0.12.1",
+    "@ceramicstudio/multiauth": "^0.2.3",
+    "caip": "^1.0.0",
```

Comment:
This is not a good idea yet afaik! We still use the old caip format.

Reply:
1.0 of caip uses the new caip10 format which ceramic doesn't support yet.

Reply:
See https://github.com/ceramicnetwork/js-ceramic/issues/1657

Reply:
See legacy format here: https://github.com/ChainAgnostic/CAIPs/blob/master/CAIPs/caip-10.md#backwards-compatibility
That's what we're using. 1.0 only supports the new format.

Reply:
thanks, rolled that back for now